src/style.css

```diff
@@ -0,0 +1,277 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  align-items: center;
+  gap: 1px;
+  width: 707px;
+  padding: 10px;
+}
+.calendar__day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  background-color: #eee;
+  height: 100px;
+  width: 100px;
+  border: 1px solid black;
+}
+.calendar__day:nth-child(1)::before {
+  content: "1";
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: block;
+}
+.calendar__day:nth-child(2)::before {
+  content: "2";
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: block;
+}
```

Comment:
Instead of manually writing each ::before pseudo-element for each day, use @for loop to generate them.

Reply:
I use in main.scss